src/makeCalculator.js

```diff
@@ -53,6 +53,10 @@ function makeCalculator() {
     },
 
     divide(value, number) {
+      if (number === 0) {
+        throw new Error();
```

Comment:
You can add in brackets a hint like: 'Not allowed to divide on zero'

Reply:
add an error description